---
float32[] left_marker_pose
float32[] right_marker_pose
float32[] middle_marker_pose
